docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -379,35 +380,6 @@ Elasticsearch Trino         Supports
 (all others)  (unsupported) (unsupported)
 ============= ============= =============
 
-Pass-through queries
```

Comment:
I am not sure if this is completely equivalent in terms of what kind of statements we can pass through ... as such we might not be able to delete the docs just yet ... maybe we should leave it until the feature is removed as well .. wdyt @martint and @kasiafi ?

Reply:
I followed the recommendation in the linked issue, but I have the same question